.github/workflows/build-test.yml

```diff
@@ -32,14 +32,15 @@ jobs:
 
     test-linux:
         needs: build
-        name: ${{ matrix.cloud }} JDBC ${{ matrix.category }} on ${{ matrix.image }}
+        name: ${{ matrix.cloud }} JDBC${{ matrix.additionalMavenProfile }} ${{ matrix.category }} on ${{ matrix.image }}
         runs-on: ubuntu-latest
         strategy:
             fail-fast: false
             matrix:
                 image: [ 'jdbc-centos7-openjdk8', 'jdbc-centos7-openjdk11', 'jdbc-centos7-openjdk17' ]
                 cloud: [ 'AWS' ]
                 category: ['TestCategoryResultSet,TestCategoryOthers,TestCategoryLoader', 'TestCategoryConnection,TestCategoryStatement', 'TestCategoryArrow,TestCategoryCore', 'TestCategoryFips']
+                additionalMavenProfile: ['', '-Dthin-jar']
```

Comment:
If we can pass empty string maybe just `mavenProfile` is better?

Reply:
there are already some profiles selected in the scripts so I prefer to name it with "additional"